### DISCUSSION TITLE
The Vimeo90k dataset downloaded from the website cannot be used directly

### DISCUSSION THREAD

**Discussion by Libyte** [open-mmlab/mmediting#1668] [Datasets] [answered]
The Vimeo90k dataset downloaded from the website cannot be used directly, still need further processing, this part of the code is not given in the project, the original dataset directory structure is as follows：
<img width="197" alt="image" src="https://user-images.githubusercontent.com/55917124/221392859-388c3995-9cb5-41c9-a315-e46cc397f517.png">

**Comment by Z-Fran** [comment]
We will update preprocess scripts in these few days.  @Libyte

**Comment by Z-Fran** [comment, marked as answer]
Hi, This [Pull Request](https://github.com/open-mmlab/mmediting/pull/1690) has updated vimeo90k preprocess scripts. @Libyte
